infra.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+services:
+    db:
+        image: mysql:5.7
+        networks:
+            - inner
+        volumes:
+            - db-data:/var/lib/mysql
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD_FILE: /run/secrets/db_root_password
+            MYSQL_DATABASE: openfire
+            MYSQL_USER: openfire
+            MYSQL_PASSWORD: testpass
```

Comment:
You hardcoded the password here? Then there's probably no point in having the password.txt files?